setup.cfg

```diff
@@ -50,6 +50,7 @@ install_requires =
     requests
     requests-kerberos
     setuptools
+    specfile
```

Comment:
+1, glad this is here so that python3-specfile will be correctly installed in worker